src/de/jost_net/JVerein/gui/menu/BuchungMenu.java

```diff
@@ -128,4 +133,38 @@ public boolean isEnabledFor(Object o)
       return false;
     }
   }
+    
+  private static class SingleGegenBuchungItem extends CheckedSingleContextMenuItem
+  {
+    private SingleGegenBuchungItem(String text, Action action, String icon)
+    {
+      super(text, action, icon);
+    }
+
+    @Override
+    public boolean isEnabledFor(Object o)
+    {
+      if (o instanceof Buchung)
+      {
+        Buchung b = (Buchung) o;
+        try
+        {
+          if ((b.getSplitId() != null) && (b.getSplitTyp() != SplitbuchungTyp.SPLIT))
+          {
+            return false;
+          }
+          if (b.getBuchungsart() != null)
```

Comment:
Sollte bei b.getBuchungsart() == null nicht direkt false zurückgegeben werden? Ohne dass "Fehler" im Log steht?

Reply:
Es wir ja false zurückgegeben. Da es keine Exception gibt läuft man nicht in das catch und es wird anschließend in Zeile 166 das false zurückgegeben.